Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Button.css';
+
+const ActionButton = props => {
+    return (
+        <button className={props.buttonStyle}>{props.text}</button>
```

Comment:
since your action button is the same style for both, you could just hardcode the classname here instead of passing it in with props, but either way works!